lib/wrappers/contributionReward.ts

```diff
@@ -587,7 +587,6 @@ export interface NewContributionProposalEventResult {
 export interface ContributionProposal {
   proposalId: Hash;
   beneficiaryAddress: Address;
-  contributionDescriptionHash: Hash;
```

Comment:
why does this get removed?

Reply:
@tibetsprague I forgot about that.  I've added it to the breaking changes above.  It was removed from the contract as part of this PR:  https://github.com/daostack/arc/pull/561/files